swap_meet/vendor.py

```diff
@@ -1,2 +1,81 @@
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+               
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+        
+    def remove(self, item):
```

Comment:
According to the instructions, this function should return `False` if the item is not in inventory. That is not happening here. The tests still pass because functions return `None` by default and the test does `assert not result`, which works for both `None` and `False`.